project.clj

```diff
@@ -4,6 +4,7 @@
    [inflections "0.9.14"]
    [dmohs/react "1.1.2+15.5.4-0"]
    [frankiesardo/linked "1.2.9"]
+   [org.broadinstitute/react-cljs-modal "20170609"]
```

Comment:
I see the benefit there, but this format is compatible with SemVer. Maybe worth the loss of readability?